src/index.ts

```diff
@@ -257,6 +258,11 @@ process.on("SIGUSR2", async () => await cleanupJobResources(jobs));
             description: "Container MAC address (e.g., aa:bb:cc:dd:ee:ff)",
             requiresArg: false,
         })
+        .option("emulate", {
+            type: "string",
+            description: "The name of a gitlab hosted runner to emulate",
```

Comment:
https://docs.gitlab.com/ee/ci/runners/hosted_runners/linux.html#machine-types-available-for-linux---x86-64

let's add the link in the description too,  for a slightly better ux.